include/battle.h

```diff
@@ -725,3 +739,5 @@ extern struct MultiBattlePokemonTx gMultiPartnerParty[3];
 extern u16 gRandomTurnNumber;
 
 #endif // GUARD_BATTLE_H
+
+#define BATTLER_MAX_HP(battlerId)(gBattleMons[battlerId].hp == gBattleMons[battlerId].maxHP)
```

Comment:
Looks like another duplicate change with PR #22 ?

Reply:
correct, had to duplicate some stuff for sturdy and tried to make the change 1:1 with #22 so it is easier to merge.